### DISCUSSION TITLE
Ip tools is not working

### DISCUSSION THREAD

**Discussion by raf-rodrigues** [postalserver/postal#1556] [Help with using Postal] [answered]
I have installed the additional ips and they are configured on the server and the postal. 

In the panel they are all configured in 100 priority. ( so that it rotates equally )

But the sending is only through the main ip of the server.

Do you have any other configuration?

**Comment by willpower232** [comment]
I think you also have to go into the organization and server that you want to tell them to make use of the IP Pool

**Reply by raf-rodrigues** [reply]
Yes, I have done that. It is already assigned to the organization. 

But it still only uses the main ip of the server. 

The additional ips are already assigned to ubuntu.

**Comment by raf-rodrigues** [comment, marked as answer]
I found the temporary solution.

I was activating Ip_tools after installing PostalHq and the ip_tools table was not being created. That's why the ips weren't being routed.

But now in a clean installation and from an initial configuration, already activating Ip Tools. A table was created. And the ips are rotating.
